vortexasdk/api/cargo_movement.py

```diff
@@ -29,6 +29,21 @@ class CargoEvent:
     end_timestamp: Optional[ISODate] = None
 
 
+@dataclass(frozen=True)
+class ParentIds:
```

Comment:
To be consistent with the rest of the ID classes naming conventions, shall we call this class `ParentID`. Also, this class refers to a single parent id, so it should be singular.